StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentOptionsViewController.swift

```diff
@@ -27,21 +37,33 @@ class VerticalSavedPaymentOptionsViewController: UIViewController {
 
     private lazy var headerLabel: UILabel = {
         let label = PaymentSheetUI.makeHeaderLabel(appearance: configuration.appearance)
-        label.text = .Localized.select_payment_method
+        let nonCardPaymentMethods = paymentMethods.filter({ $0.type != .card })
+        label.text = nonCardPaymentMethods.isEmpty ? .Localized.select_card : .Localized.select_payment_method
         return label
     }()
+    
+    private lazy var paymentMethodRows: [(paymentMethod: STPPaymentMethod, button: PaymentMethodRowButton)] = {
```

Comment:
Not sure how I feel about this. It's tricky to associate a `STPPaymentMethod` with a row button when:
1. Not using a table view
2. Not muddying up the `PaymentMethodRowButton.ViewModel` with an `STPPaymentMethod`.

Open to suggestions

Reply:
I don't have strong ideas either. Why not have the payment method as a property of the view (either in the viewmodel or just directly)?

Reply:
I just felt like the `PaymentMethodRowButton` and `PaymentMethodRowButton.ViewModel` shouldn't know about an entire `STPPaymentMethod` and it should just know the data it needs to draw itself, no other reason.